partner_contact_type_end_user/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Contact Type End User',
```

Comment:
As this is a completely new module, please apply black conventions on them, easing later migration to 13.0.